packages/@okta/vuepress-site/docs/guides/build-api-integration/main/index.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Build an API service integration
+excerpt: Learn how to build and register an API service integration to the Okta Integration Network.
+layout: Guides
+---
+
+<ApiLifecycle access="ea" />
+
+This guide explains how to build and submit an API service integration to the Okta Integration Network (OIN) for review and publishing.
+
+---
+
+**Learning outcomes**
+
+* Learn how to build an API service integration that can be published in the OIN.
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+   > **Note:** Your org needs to have the API Service Integration feature enabled. Contact your Okta account team to enable the feature.
+* A service app that needs to access Okta APIs for your customer
+
+
+---
+
+## Overview
+
+A service-to-service app where where a backend service or a daemon calls [Okta management APIs](/docs/reference/core-okta-api/) for a tenant (Okta org) can be published in the Okta Integration Network (OIN) as an API service integration.
+API service integrations access Okta APIs using the OAuth 2.0 Client Credentials grant flow, where access is not associated with a user and resources can be restricted with scoped access tokens. Each access token enables the bearer to perform specific actions on specific Okta endpoints, with that ability controlled by which scopes the access token contains.
+
+> **Note:** Currently, Okta only supports OAuth APIs listed in [Scopes and supported endpoints](/docs/guides/implement-oauth-for-okta/main/#scopes-and-supported-endpoints). We're working towards supporting scopes for all Okta API endpoints.
+
+If you have an API service integration for your app in the OIN, your customers can configure your integration to implement the OAuth 2.0 Client Credentials flow with their Okta tenant org. Each customer Okta org has its own authorization server. When a customer authorizes your API service integration to access their org, Okta generates a unique set of credentials (client ID and client secret) for that org.
+You must collect and store these credentials for each customer to allow your integration to work with your customer's Okta org.
+
+### API service integration Client Credentials grant flow
+
+At a high-level, the OAuth 2.0 Client Credentials flow for an API service integration has the following steps:
+
+1. Your customer's instance of your service app makes an authorization request to their Okta Authorization Server using their client credentials.
```

Comment:
I guess technically they could link to our catalog or a page in admin where the customer click's authorize, but so far we're expecting the customer to start in Okta and do the authorizing there before the ISV does anything. So they technically don't need to do an authorize request like they do with auth code or PKCE. ISV gets the credentials and then jumps right to the token request.

Reply:
I am assuming that ISV requests for access token each time ONE Okta API request is required. Let me know if you want to say anything about the token usage or expiry time?

Reply:
Right now, the token expires after an hour. So they can make API calls until then. I'm not sure how the Fed team feels about us talking about the hour expiry. They might be trying to reserve the right to change that in the future? So looking at the expiry time in the token response as you have it is the best practice.

They can also put it in a try/catch type syntax. If the token is expired, fetch a new one.

Reply:
My comment here is just that the ISV doesn't need to make an authorization request. Customer authorizes in Okta and then they just make a token request.

Reply:
Updated this section to remove "authorization request" and use "access token request" instead. Thanks!